### DISCUSSION TITLE
Better import dupe handling on similar but not identical albums?

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#4934] [General]
Adrian and I had some discussion based around a collision between two albums marked as dupe when they were actually unique.

https://github.com/beetbox/beets/issues/3916

I think Adrian may have misunderstood the situation - I believe the distance_weights wouldn't have helped here, because my tags were not that thorough. But once I decided to import, the duplicate detection was faulty - it presumably only had Stephen Schwartz and Godspell to go off of. I wonder if the "albumartist album" default keys would lead to bad results when importing multiple albums from groups like Weezer who re-use the same album name constantly. If we assume every album has a valid MBID, is it as simple as using release group MBID in the duplicate keys? See e.g. [Green](https://musicbrainz.org/release-group/c7b245c9-8099-32ea-af95-893acedde2cf) and [Blue](https://musicbrainz.org/release-group/923d5ba6-7eee-3bce-bcb2-c913b2bd69d4).

**Comment by sampsyo** [comment]
Yeah, we could conceivably use the release group ID (not the release ID, of course) _when both the existing and imported album have MBIDs attached_. We probably wouldn't want to do this when only one has an ID.
